features/note_tags.feature

```diff
@@ -0,0 +1,17 @@
+Feature: User tags notes
+
+  Scenario: User creates a note
+    Given I have an existing user account
```

Comment:
^^Agreed. After observing Matts tests, I can see how this could be reduced. But this is great!